Automattic/jetpack
# [Earn] Make Earn products free for all Jetpack plans

Relates to https://github.com/Automattic/gold/issues/11

## Proposed changes:

Make Earn products free for all Jetpack plans.  Follow up from https://github.com/Automattic/jetpack/pull/30398

### Other information:


## Jetpack product discussion


p1HpG7-lSW-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:
- Run `bin/jetpack-downloader test jetpack add/make-earn-products-free-for-all` on your sanbox
- Sandbox public-api.wordpress.com and subscribe.wordpress.com
- Either create a site via WPCOM (and sanbox it), or use jetpack-beta.
- Connect Jetpack free version
- Create a donation block in a post
- Test it
Result:
https://total-antelope.jurassic.ninja/2023/05/08/donate/

See my videos:

https://user-images.githubusercontent.com/790558/236858249-fff8db35-9be5-4ea7-bd5d-faf4949ad4cf.mov


https://user-images.githubusercontent.com/790558/236857412-77ef764a-d761-4fe1-aaf3-5cfa44bb6373.mov
